invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -448,13 +527,22 @@ def calc_text_size(self, text, font=None):
 
         w = 0
         h = 0
-        for t in text.split('\n'):
+        for t in text.split("\n"):
             _w, _h = gc.GetTextExtent(t)
             w = max(w, _w)
             h += _h
         return w, h
 
-    def draw_line(self, pos0, pos1, arrow_start=False, arrow_end=False, colour=(255, 0, 0, 128), width=2, style=wx.SOLID):
+    def draw_line(
+        self,
+        pos0,
+        pos1,
```

Comment:
Tuple of of 2 integers

Reply:
I think pos1 is also a`Tuple[float, float]`